### DISCUSSION TITLE
Remove composite/drawing after it has been added?

### DISCUSSION THREAD

**Discussion by rgdcastro** [dlemstra/Magick.NET#1664] [Help] [answered]
Hi there,

I just wanted to check if anyone knows if it's possible to remove a composite/drawing once it has been added?

What I am trying to do is the following,

1. Open an image once.
2. Add composite such as text, thumbnail, line drawing, etc...
3. Save the image.
4. Remove the composite everything added on step 2.
5. Go back to step 2 until everything is done.

Basically, what I am trying to avoid is the penalty of re-opening the image every time. I can probably create another Image object which always hold the unmodified image but wanted to check first if my question is possible since it will probably consume less memory especially if the image is huge.

**Comment by dlemstra** [comment, marked as answer]
You will need to maintain a copy of the image if you want to do that. There is no way to go back to the original. You will need to create a `.Clone()` of the input image.
